include/avif/avif.h

```diff
@@ -400,8 +400,8 @@ enum
     AVIF_MATRIX_COEFFICIENTS_CHROMA_DERIVED_CL = 13,
     AVIF_MATRIX_COEFFICIENTS_ICTCP = 14,
 #if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
-    AVIF_MATRIX_COEFFICIENTS_YCGCO_RE = 15,
-    AVIF_MATRIX_COEFFICIENTS_YCGCO_RO = 16,
+    AVIF_MATRIX_COEFFICIENTS_YCGCO_RE = 16,
```

Comment:
Optional: It would be good to add a comment on what value 15 is. But I see value 3 is also missing and we don't have a comment for that. So it should be fine to not add a comment.

Reply:
Are 3 and 15 both "Reserved" in the latest edition?

Reply:
I checked H.273 (09/23). 3 is reserved, and 15 is IPT-C2.